src/navigation/TabBarNavigator.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FC, useCallback, useMemo } from 'react'
+import { SvgProps } from 'react-native-svg'
+import {
+  BottomTabBarProps,
+  createBottomTabNavigator,
+} from '@react-navigation/bottom-tabs'
+import { Edge } from 'react-native-safe-area-context'
+import NavBar from '../components/NavBar'
+import Dollar from '../assets/images/dollar.svg'
+import Gem from '../assets/images/gem.svg'
+import Swaps from '../assets/images/swaps.svg'
+import Transactions from '../assets/images/transactions.svg'
+import Notifications from '../assets/images/notifications.svg'
+import { Color } from '../theme/theme'
+import HomeNavigator from '../features/home/HomeNavigator'
+import CollectableNavigator from '../features/collectables/CollectablesNavigator'
+import ActivityNavigator from '../features/activity/ActivityNavigator'
+import NotificationsNavigator from '../features/notifications/NotificationsNavigator'
+import SafeAreaBox from '../components/SafeAreaBox'
+import Box from '../components/Box'
+import useEnrichedTransactions from '../utils/useEnrichedTransactions'
+
+const Tab = createBottomTabNavigator()
+
+function MyTabBar({ state, navigation }: BottomTabBarProps) {
+  const { hasNewTransactions, resetNewTransactions } = useEnrichedTransactions()
+  // TODO: Verify if it is even worth getting all pending rewards considering the amount of account fetches required for every hotspot.
```

Comment:
Who can make this decision?

Reply:
I think we can. Problem is we cant fetch pending rewards for all hotspots without making multiple queries. I was going to bring this up on the next PR when the compressed hotspot NFTs are ready to go

Reply:
Will address these TODOs with compressed NFTs